### PR TITLE
feat(linter) eslint-plugin-next google-font-display

### PR DESCRIPTION
**Pull request by camc314**

### CODE REVIEW

**Comment by camc314** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1931** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1931?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #1932** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1932?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #1933** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1933?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #1934** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1934?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/oxc-project/oxc/1931?utm_source=stack-comment).

File: crates/oxc_linter/src/rules/nextjs/google_font_display.rs [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+declare_oxc_lint!(
+    /// ### What it does
+    ///
+    ///
+    /// ### Why is this bad?
+    ///
+    ///
+    /// ### Example
+    /// ```javascript
+    /// ```
```
Comment (Dunqing) [inline crates/oxc_linter/src/rules/nextjs/google_font_display.rs:47]: We need to write something here

Comment (Boshen) [reply crates/oxc_linter/src/rules/nextjs/google_font_display.rs:47]: Fix this later 😅

**Comment by Boshen** [comment]
### Merge activity

* **Jan 7, 10:49 PM**: @Boshen started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/oxc-project/oxc/1931).

**Pull request merged**